src/components/TodoForm.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { useTodoContext } from '../context/TodoContext';
+import { addTodo } from '../api/todos';
+import { Error } from '../types/Error';
+
+export const TodoForm: React.FC = () => {
+  const {
+    setTodos,
+    setError,
+  } = useTodoContext();
+
+  const [todoTitle, setTodoTitle] = useState('');
+  const [isCreating, setIsCreating] = useState(false);
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
```

Comment:
Clear the error state when the input is valid. Add 'setError(null);' before 'setIsCreating(true);'